src/Backend/API.ts

```diff
@@ -49,6 +50,7 @@ function backendRegistrationApi() {
         globalExecutorArray.push(executor);
       }
       Logger.info(logTag, 'Backend', backendName, 'was registered into ONE-vscode.');
+      vscode.commands.executeCommand('one.device.refresh');
```

Comment:
(Q) Do you consider the timing issue? Does it happen frequently?

Reply:
>Do you consider the timing issue?

I add this for timing issue. if this register API call start after activate this extension.

>Does it happen frequently?

sometimes occur. but I want to prevent this.

Reply:
There was a discussion from https://github.com/Samsung/ONE-vscode/pull/1101#issuecomment-1195099002. For now, we concluded by adding comment about it. 

Could you please add a todo comment like this commit? 2d5814350a9e1525a052c6593b2a8bd3ebe971a8 (maybe NOTE part isn't needed..)